### PR TITLE
Remove useless sentry

### PR DESCRIPTION
**Pull request by FabianDevel**
SentryLog.e() captures error on its own

We don't want that in this places

### CODE REVIEW

File: app/src/main/java/com/infomaniak/mail/utils/LoginUtils.kt [hunk 1]
```diff
@@ -123,7 +123,7 @@ class LoginUtils @Inject constructor(
         infomaniakLogin.deleteToken(
             okHttpClient = HttpClient.okHttpClientNoTokenInterceptor,
             token = apiToken,
-            onError = { SentryLog.e("DeleteTokenError", "API response error: $it") },
+            onError = { SentryLog.i("DeleteTokenError", "API response error: $it") },
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/utils/LoginUtils.kt:126]: See with @LunarX for this one.

File: app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt [hunk 2]
```diff
@@ -78,7 +78,7 @@ class LogoutUser @Inject constructor(
         appContext.getInfomaniakLogin().deleteToken(
             okHttpClient = HttpClient.okHttpClientNoTokenInterceptor,
             token = apiToken,
-            onError = { SentryLog.e("DeleteTokenError", "API response error: $it") },
+            onError = { SentryLog.i("DeleteTokenError", "API response error: $it") },
```
Comment (KevinBoulongne) [inline app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt:81]: See with @LunarX for this one.

**Review by KevinBoulongne — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/infomaniak/mail/utils/LoginUtils.kt (hunk 1 above)
Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/mail/utils/LoginUtils.kt]: Moved to https://github.com/Infomaniak/android-kMail/pull/2037

File: app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt (hunk 2 above)
Comment (FabianDevel) [reply app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt]: Moved to https://github.com/Infomaniak/android-kMail/pull/2037

**Review by KevinBoulongne — APPROVED** [review]

**Pull request merged**